packages/manager/modules/dbaas-logs/src/logs/detail/aliases/home/logs-aliases-home.html

```diff
@@ -21,86 +24,78 @@ <h3 data-translate="logs_aliases_title"></h3>
         </oui-datagrid-topbar>
         <oui-datagrid-column
             title="::'logs_aliases_col_title' | translate"
-            property="info.name"
+            property="name"
             sortable="asc"
             type="string"
             searchable
             filterable
         ></oui-datagrid-column>
         <oui-datagrid-column
             title="::'logs_aliases_col_description' | translate"
-            property="info.description"
+            property="description"
             sortable
             type="string"
             searchable
             filterable
```

Comment:
Sames as : https://github.com/ovh/manager/pull/9310/files#r1203778368
```suggestion
            data-filterable
```